src/sentry/integrations/github/integration.py

```diff
@@ -305,6 +306,10 @@ def post_install(
                 }
             )
 
+        link_all_repos.apply_async(
+            kwargs={"integration_id": integration.id, "organization_id": organization.id}
+        )
```

Comment:
Is this meant to be deployed immediately? Just wondering if there should be a feature flag here

Reply:
yes immediately

Reply:
@leeandher this would only affect orgs installing github after we deploy the change. So I don't expect a flurry of new repo linking immediately